apps/wallet/src/ui/app/wallet/locked-page/index.tsx

```diff
@@ -78,7 +78,7 @@ export default function LockedPage() {
                             }}
                         >
                             {({ touched, errors, isSubmitting, isValid }) => (
-                                <Form className={st.form}>
```

Comment:
just update the scss module instead of changing it to tailwind and just add the top margin there if necessary.

Reply:
done

Reply:
> @Jibz1 just noticing a copy gaffe in Figma. Apologies. (Just updated Figma with this detail.)
> 
> Would you please update the copy for both the paragraph and the checkbox line? <img alt="image" width="461" src="https://user-images.githubusercontent.com/109102528/211934884-c36501df-353d-4a78-9629-5b3002b9520a.png">

done